pkg/logs/internal/processor/processor.go

```diff
@@ -79,6 +79,7 @@ func (p *Processor) run() {
 	for msg := range p.inputChan {
 		p.processMessage(msg)
 		p.mu.Lock() // block here if we're trying to flush synchronously
+		//nolint:staticcheck
```

Comment:
Can you add a description of what this code is doing, to justify the `nolint`?

Reply:
My thought was that the above comment on line 81 explains what's going on here. Any suggestions on how to make this clearer?

Reply:
ah, didn't notice that comment. I guess it's okay like this then, I can't think of a better comment